src/domain/app/footer/Footer.tsx

```diff
@@ -41,6 +41,10 @@ const Footer = ({ isMobileMenuOpen = false }: Props): React.ReactElement => {
               <Link href="/accessibility">
                 {t('common:accessibilityStatement.title')}
               </Link>{' '}
+              •{' '}
+              <a href={FEEDBACK_LINKS[locale]} target="_blank" rel="noreferrer">
```

Comment:
should we create a custom components for external link with preconfigured attributes?

Reply:
That is an option but I can't see many more of these links coming so is there a big benefit? So for time saving (tight budget) should we go with this?

Reply:
I thought this more and maybe it would be good to have Compoment for external links... It could also always put helper text for screen readers.